### PR TITLE
Fixes #751: "require chart.js is not defined" error removed.

### LINKED ISSUE #751: remove the require is not defined error on susper
**Actual Behaviour**

screen shot 2017-08-15 at 2 24 16 pm

### PR DESCRIPTION
**Pull request by harshit98**
Fixes issue #751 

Changes: 
- Fixed the error which was being thrown by chart.js

Demo Link: https://harshit98.github.io/susper.com/

Please review @nikhilrayaprolu @Marauderer97

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
   <link rel="stylesheet" href="assets/css/bootstrap-base.css">
   <!-- jQuery library -->
   <script src="//ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
-  <script src="assets/chart.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.4.0/Chart.bundle.min.js"></script>
```
Comment (nikhilrayaprolu) [inline src/index.html]: it is good if we could serve this from our local system,please download the chart.bundle.min.js , save it locally and serve that.

Comment (harshit98) [inline src/index.html]: @nikhilrayaprolu I'm not able to find `chart.bundle.min.js` file. Only CDN link was available.

Comment (nikhilrayaprolu) [inline src/index.html]: you can download it from the cdn link and place it offline

**Comment by harshit98** [comment]
Done @nikhilrayaprolu

**Pull request merged**